vimeo-networking/src/main/java/com/vimeo/networking/model/error/VimeoError.java

```diff
@@ -60,7 +61,13 @@
     protected String mDeveloperMessage;
 
     @SerializedName("error_code")
-    protected ErrorCode mErrorCode;
+    private String mRawErrorCode;
+
+    @NotNull
+    private ErrorCode mErrorCode = ErrorCode.DEFAULT;
+
+    @NotNull
+    private LocalErrorCode mLocalErrorCode = LocalErrorCode.DEFAULT;
```

Comment:
Should this be nullable? We'll probably only want to log local error codes if we see they have a value, and a null check seems more intuitive to me than not logging if it shows up as "default", but logging otherwise.

The alternative would be adding documentation to say that a value of "DEFAULT" means there is no "LocalError".

Reply:
The previous behavior was not assigning the `DEFAULT` value to the enum in the case where the value was unrecognized, instead it would assign it to null. I looked at usage of this property and it is being assumed to be not null and it is assumed that `DEFAULT` is being assigned if no code is present. For those reasons, I decided to change it to not null and assign it a default value. 

Alternately, I can remove the default and `null` can be used instead. However, making it non null and giving it a default value makes it easier for consumers to switch on the value.

Reply:
I think either is fine, but whatever you choose it should just be clearly documented.

Reply:
I added a comment to the getter of `ErrorCode` and `LocalErrorCode` documenting that if no value was set, then the value will be set to `DEFAULT`.